github/ci/services/prometheus-stack/manifests/stack/production-e2e-workloads/prometheus-federation.yaml

```
@@ -0,0 +1,21 @@
+---
+# PodMonitor to scrape prometheus metrics from each ephemeral Prometheus instance of each prow-job kubevirtci cluster
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: prometheus-stack-federation
+  namespace: default
```

Maybe better to keep it under the monitoring namespace, wdyt?
```suggestion
  namespace: monitoring
```

---

It might make sense to create it in `kubevirt-prow-jobs` ns instead, but I think that something more scoped than `default` would be better, let me know wdyt.

---

I think that `monitoring`, like you suggested is a good namespace for it.

---

Updated!